### PR TITLE
Prepare for release 

### PR DESCRIPTION
**Pull request by Evelyn-M**
This is a preparation for the next release (v1.2).

### CODE REVIEW

File: src/osm_flex/clip.py [hunk 1]
```diff
@@ -160,19 +160,23 @@ def _build_osmconvert_cmd(shape, osmpbf_clip_from, osmpbf_output):
         file path to planet.osm.pbf or other osm.pbf file to clip
     osmpbf_output : str or pathlib.Path
         file path (incl. name & ending) under which extract will be stored
+        
+    Note
+    -----
+    If cross-border multi-polygons or lines should be retained, comment out
+    below options '--complete-ways' and '--complete-multipolygons'. This only
+    works with parent files < 2GB.
```
Comment (peanutfun) [inline src/osm_flex/clip.py:168]: This only works for source installations, but not if the package is loaded from PyPI. Maybe add this as an optional parameter to the function? Something like
```python
def _build_osmconvert_cmd(shape, osmpbf_clip_from, osmpbf_output, complete=False):
    pass
    """
    Set ``complete=True`` to retain cross-border multi-polygons and lines.
    """
```

File: tests/test_simplify.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            geometry = [
+                point1,
+                line1,
+                polygon1,
+            ])
+        
+        # get rid of polygon2
+        gdf_removed = remove_small_polygons(gdf_small_poly, 0.5)
+        
+        self.assertTrue((gdf_removed == gdf_no_small_poly).all().values[0])
```
Comment (peanutfun) [inline tests/test_simplify.py:57]: Comparing (Geo)DataFrames can be done more elegantly with [pandas.testing.assert_frame_equal](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.testing.assert_frame_equal.html#pandas.testing.assert_frame_equal)
```python
import pandas.testing as pdt

pdt.assert_frame_equal(gdf_removed, gdf_no_small_poly)
```
(same for all tests in this module)

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,23 @@
 # Changelog
+## v1.2.0
```
Comment (peanutfun) [inline CHANGELOG.md:2]: Current version is 1.0.1, so the next minor release should be
```suggestion
## v1.1.0
```

**Review by peanutfun — CHANGES_REQUESTED** [review]
Looks good, just a few suggestions. Also, you still need to set the new version in `pyproject.toml`. Then it's ready to go!

File: src/osm_flex/clip.py (hunk 1 above)
Comment (peanutfun) [reply src/osm_flex/clip.py:168]: I think adding something like `kernel_kwargs`, and documenting which options are available for each kernel, would be fine. But maybe this is an idea for the next release 🙃

**Review by peanutfun — APPROVED** [review]
Excellent

File: src/osm_flex/clip.py (hunk 1 above)
Comment (Evelyn-M) [reply src/osm_flex/clip.py:168]: Ah yes, I didn't think of this possibility. Good point. Will make a note.

**Pull request merged**